cherrypy/_cprequest.py

```diff
@@ -911,6 +912,9 @@ def collapse_body(self):
         self.body = newbody
         return newbody
 
+    def __str__(self):
```

Comment:
Oops.. You got me wrong. This magic method here will cause `str(response)` to return the UUID, breaking representation of the response itself. I meant you need to assign some lazy object as request's attribute from within `_cplogging` module.